src/icon.js

```diff
@@ -1,8 +1,17 @@
 const path = require('path');
 
-module.exports = {
-  done: path.join(__dirname, 'icons/macos/doneTemplate.png'),
-  todo: path.join(__dirname, 'icons/macos/todoTemplate.png'),
-  load: path.join(__dirname, 'icons/macos/loadTemplate.png'),
-  fail: path.join(__dirname, 'icons/macos/failTemplate.png')
-};
+if (process.platform == 'win32') {
```

Comment:
Again, strict equality checks please! Your editor/IDE should be complaining about this if it has eslint integration.